docs/kilt-baseline-replication.md

```diff
@@ -0,0 +1,136 @@
+# Replication guide KILT
+
+## Setup env
+
+```bash
+# Create a virtual env
+conda create -n kilt37 -y python=3.7 && conda activate kilt37
+
+# Get pyserini scripts and install package
+git clone https://github.com/castorini/pyserini.git
+pip install pyserini
+
+# Get KILT scripts and install package
+git clone https://github.com/facebookresearch/KILT.git
```

Comment:
Can you clarify the directory structure where you want to put everything?

I think it might make sense to have everything under `pyserini/`? For example, the KILT corpus can go into `collections/`?